apps/omg_watcher_info/test/omg_watcher_info/db/block_test.exs

```diff
@@ -32,6 +32,31 @@ defmodule OMG.WatcherInfo.DB.BlockTest do
     end
   end
 
+  describe "get/1" do
+    @tag fixtures: [:initial_blocks]
+    test "retrieves a block by block hash" do
+      hash = "##{1000}"
+      block = hash |> DB.Block.get()
+
+      assert block.__struct__ == DB.Block
+      assert block.hash == hash
+    end
+
+    @tag fixtures: [:initial_blocks]
+    test "the transactions associated with a block can be preloaded" do
+      blknum = 1000
+
+      %{:transactions => transactions} =
+        DB.Repo.get(DB.Block, blknum)
+        |> DB.Repo.preload(:transactions)
+
+      hash = "##{blknum}"
+      block = hash |> DB.Block.get()
```

Comment:
```suggestion
      block = DB.Block.get(hash)
```

Reply:
Thanks @InoMurko - this test was removed as pre-loading transactions no longer needed.